pkgs/top-level/all-packages.nix

```diff
@@ -12689,6 +12689,16 @@ with pkgs;
 
   upstart-check-config = callPackage ../os-specific/linux/upstart/check-config.nix {};
 
+  usbguard = usbguard_libgcrypt;
+
+  usbguard_libgcrypt = libsForQt5.callPackage ../os-specific/linux/usbguard {
+    libsodium = null;
```

Comment:
The only advantage I saw was to be able to choose. One of them can be removed without loosing any functionality. IIRC they are only used for hashing.